packages/fcl/src/wallet-utils/__snapshots__/encode-provable-authn-signing.test.js.snap

```diff
@@ -1,5 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`encode provable authn encode provable authn with appTag 1`] = `"464c4f572d56302e302d75736572000000000000000000000000000000000000f850b8403436346334663537326434613533326435333434346230303030303030303030303030303030303030303030303030303030303030303030303030303030303086abc123def45686017c05815137"`;
+exports[`encode provable authn encode provable authn with appTag 1`] = `"464c4f572d56302e302d75736572000000000000000000000000000000000000f1a0464c4f572d4a532d53444b000000000000000000000000000000000000000000880000abc123def45686017c05815137"`;
 
-exports[`encode provable authn encode provable authn without appTag 1`] = `"464c4f572d56302e302d75736572000000000000000000000000000000000000ce86abc123def45686017c05815137"`;
+exports[`encode provable authn encode provable authn without appTag 1`] = `"464c4f572d56302e302d75736572000000000000000000000000000000000000d0880000abc123def45686017c05815137"`;
```

Comment:
Hmm, so it looks like this change ended up being breaking. Any idea why?

Are you confident that this new version is correct as per the spec? I think it's OK to break it as long as we're confident that this is the format we want going forward.

Reply:
@psiemens I second your hmm. Snapshot is different I think due to addition of address buffer and switch to sansPrefix. If there is alignment between signing/verifying I don't believe it's a breaking change. Integration testing succeeds in verifying using previous encode-signing with new encode-verifying and vice versa.
What spec are you referring to?

Reply:
I think I found the discrepancies.

Old encoding:
- `USER_DOMAIN_TAG`: `464c4f572d56302e302d75736572000000000000000000000000000000000000`
-  `APP_DOMAIN_TAG`: `50b84034363463346635373264346135333264353334343462303030303030303030303030303030303030303030303030303030303030303030303030303030303030`
- `ADDRESS`: `abc123def456`
- `TIMESTAMP`: `86017c05815137`

New encoding:
- `USER_DOMAIN_TAG`: `464c4f572d56302e302d75736572000000000000000000000000000000000000`
- ⚠️ `APP_DOMAIN_TAG`: `a0464c4f572d4a532d53444b000000000000000000000000000000000000000000`
- ⚠️ `ADDRESS`: `0000abc123def456`
- `TIMESTAMP`: `86017c05815137`

Here's what changed:
- `APP_DOMAIN_TAG` was previously being encoded as a string (rather than bytes) and contains a bunch of extra unicode bytes. I wondered why this was, and found [this line in the RLP encoder](https://github.com/onflow/fcl-js/blob/master/packages/rlp/src/index.js#L210). The encoder only treats a hex string as a byte array _if it has a 0x prefix_.
- `ADDRESS` was previously not being left-padded to 8 bytes (i.e. 16 hex characters). We want this left padding to happen so that this encoding is consistent with how addresses are encoded in transactions.

I think this new encoding format is correct and much easier to replicate across other client libraries. 👍